language/resource.language.pt_br/strings.po

```diff
@@ -0,0 +1,2229 @@
+
+msgctxt "#31294"
+msgid "Press down for bookmarks and chapters"
+msgstr "Pressione para baixo para marcadores e capítulos"
+
+msgctxt "#31295"
+msgid "Press down for channels"
+msgstr "Pressione para baixo para canais"
+
+msgctxt "#31296"
+msgid "Press down for playlist"
+msgstr "Pressione para baixo para ver a lista de reprodução"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31297"
+msgid "Extra fanart"
+msgstr "Fanart extra"
+
+msgctxt "#31298"
+msgid "Press down for movies and tv shows starring this actor"
+msgstr "Pressione para ver filmes e séries estrelados por esse ator"
+
+msgctxt "#31299"
+msgid "Information is loading"
+msgstr "As informações estão sendo carregadas"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31300"
+msgid "Customise window background"
+msgstr "Personalizar fundo da janela"
+
+msgctxt "#31302"
+msgid "Management options"
+msgstr "Opções de gerenciamento"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31303"
+msgid "Customise fallback image folder"
+msgstr "Pasta de imagem substituta personalizada"
+
+#: /1080i/Includes_DialogInfo.xml
+msgctxt "#31307"
+msgid "Instrument"
+msgstr "Instrumento"
+
+#: /1080i/Includes_DialogInfo.xml
+msgctxt "#31308"
+msgid "Mood"
+msgstr "Humor"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31309"
+msgid "Glass"
+msgstr "Vidro"
+
+#: /1080i/Custom_1161_Dialog_Weather.xml
+msgctxt "#31310"
+msgid "Expand"
+msgstr "Expandir"
+
+#: /1080i/Custom_1161_Dialog_Weather.xml
+msgctxt "#31311"
+msgid "Weekly"
+msgstr "Semanalmente"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31312"
+msgid "Spinner"
+msgstr "Girador"
+
+#: /1080i/Custom_1116_Dialog_Shortcuts.xml
+msgctxt "#31313"
+msgid "Always visible"
+msgstr "Sempre visível"
+
+#: /1080i/Includes_MediaFilter.xml
+msgctxt "#31319"
+msgid "View"
+msgstr "Exibir"
+
+#: /1080i/Includes_Views.xml
+msgctxt "#31320"
+msgid "Square"
+msgstr "Quadrado"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31321"
+msgid "Configure viewtypes"
+msgstr "Configurar tipos de exibição"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31322"
+msgid "Age"
+msgstr "Idade"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31324"
+msgid "Content"
+msgstr "Conteúdo"
+
+#: /1080i/Includes_Shortcuts_Options.xml
+msgctxt "#31327"
+msgid "Choose category spotlight"
+msgstr "Escolher o destaque da categoria"
+
+#: /1080i/Custom_1116_Dialog_Shortcuts.xml
+msgctxt "#31328"
+msgid "Edit widgets"
+msgstr "Editar widgets"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31329"
+msgid "Text"
+msgstr "Texto"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31330"
+msgid "Crop"
+msgstr "Cortado"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31331"
+msgid "Background style"
+msgstr "Estilo de fundo"
+
+#: /1080i/Includes_Search_Filter.xml
+msgctxt "#31332"
+msgid "New search"
+msgstr "Nova pesquisa"
+
+msgctxt "#31334"
+msgid "Comments"
+msgstr "Comentários"
+
+#: /1080i/GameOSD.xml
+msgctxt "#31335"
+msgid "Game menu"
+msgstr "Menu de Game"
+
+#: /1080i/GameOSD.xml
+msgctxt "#31336"
+msgid "Select + X"
+msgstr "Selecionar + X"
```

Comment:
I'm not sure if Select should be translated as it's a game button and they generally don't seem to be translated on the controllers.

e.g. See the Brazilian Playtronic NES controllers
https://http2.mlstatic.com/D_NQ_NP_2X_691010-MLB75462072085_032024-F.webp